lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -83,9 +83,9 @@ class S1ApUtil(object):
     lib_name = "libtfw.so"
 
     _cond = threading.Condition()
-    _msg = Queue()
-    # Default maximum wait time is 60 sec (1 min)
-    MAX_RESP_WAIT_TIME = 60
+    _msg: Queue = Queue()
+    # Default maximum wait time is 180 sec (3 min)
```

Comment:
Do we need this change or is any testcase failing because of the old max response timeout value of 60 seconds?

Reply:
Yes, it makes the tests more stable.